op-service/metrics/balance.go

```diff
@@ -0,0 +1,61 @@
+package metrics
+
+import (
+	"context"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+// 10^18
+var etherScalar = new(big.Int).Exp(big.NewInt(10), big.NewInt(18), nil)
```

Comment:
This constant is defined in many places throughout the monorepo and also in geth itself

Reply:
Do you know the names of the other definitions? There's one with the same name (also private) in the op-e2e package. In geth there is a private one in `cmd/faucet`.

Reply:
e2eutils or action testing has one too iirc. We should probably just put a global in the `op-node/eth` package

Reply:
https://github.com/ethereum/go-ethereum/blob/b628d7276624c2d8ea7dd97d2259a2c2fce7d3cc/params/denomination.go

Reply:
Oh I see, it's 1E18 there. I'm probably gonna use that + `big.NewRat` rather than going through `big.Ints` here.